11-Backbonejs/07-JavaScript-Objects-to-Backbone-Models.md

```diff
@@ -0,0 +1,230 @@
+# Scrabble-JS Total Conversion to Backbone.js!
+
+We've already done Scrabble & Player objects in Javascript.  Now we'll take those generic objects and turn'em into Backbone Models! 
+
+## Learning Goals
+With this live code/lecture you should develop the following skills:
+- Learn how Backbone Models compare with pure Javascript Objects.
+- Learn how to adjust Jasmine tests for Backbone models
+
+
+### Baseline Setup
+The starting code with Scrabble & Player objects are located [here](https://github.com/CheezItMan/scrabble-js-to-backbone).  The file has two JavaScript files `src/scrabble.js` and `src/player.js`.  It also has two spec files `spec/scrabble_spec.js` and `spec/player_spec.js`.  
+
+Go ahead and fork & clone the repo.  
+
+Then take a minute to look at the `scrabble.js` and `player.js` together with your partner to get a bit familiar with them.  
+
+#### Commands
+Because this uses the scaffold you still need to run `npm install` after forking and cloning the repository. **You do not need to run** `npm start` as we won't have a web application.
+
+To run your tests you should use the command `npm test`.
+
+#### Using the REPL
+If you want to interact with the Scrabble objects in the terminal via a REPL, you'll need to do things in a specific way (due to the scaffold's use of webpack).
+
+To access the REPL you should use the command `npm run repl` in your project directory. Once in the REPL, you'll need to use the following syntax to import your file (this example assumes it was named `src/scrabble.js`):
+
+```javascript
+const Scrabble = require('scrabble').default;
```

Comment:
I thought we had it in the original intro to JS resource, but apparently I misremembered since the only (current) references to it are in the separate resource just for that.

We should probably consider whether to make them the main thing we teach in the JS curriculum for C7 because [let](http://caniuse.com/#search=let) and [const](http://caniuse.com/#search=const) have 87% and 98% availability amongst browsers used in the U.S. at the moment, respectively.